### ISSUE TITLE
Is it possible to run a config command using FastCli?

### ISSUE THREAD

**Issue by theotherguy2175** [GADify/ZTP-Scripts#1]
Are you able to run a configuration command like "hostname" using the fastcli from bash?

**Comment by GADify** [comment]
Yes, but that's not what this script is for.

**Comment by theotherguy2175** [comment]
Right, sorry. I'm just having a hard time finding any information on it. And thought this would be a good place to ask.

**Comment by GADify** [comment]
Get yourself a copy of Arista Warrior. :)

The key that's tough to grok until someone shows you is that you must send all the commands at once, so you can't send "conf" and "hostname foo" as two commands. Plus you must specify priv level. Try cli -p 15 -c 'conf\nhostname foo'. The \n is a newline that separates the commands.  Note that cli and fastcli commands are the same thing now and have been for many years.

**Comment by theotherguy2175** [comment]
I do have a copy. It's a great book! That's why I came here hoping to get answers. Running the command on bash works just fine.

The problem I'm running into is trying to use that on the subprocess library in python. It doesn't like:

`subprocess.Popen(["/usr/bin/FastCli", "-p15", "-c", "$'conf\n hostname foo'"], stdout=subprocess.PIPE)`

Nor does it like:

`subprocess.Popen(["/usr/bin/FastCli", "-p15", "-c", r"$'conf\n hostname foo'"], stdout=subprocess.PIPE)`

**Comment by theotherguy2175** [comment]
I was actually just able to make a work around by using the os library:

`import os`
`command = "FastCli -p15 -c $'conf\n hostname foo'"`
`os.system(command)`
